prism/prism.py

```diff
@@ -4,14 +4,15 @@
 from web_fragments.fragment import Fragment
 from xblock.core import XBlock
 from xblock.fields import Scope, String, Integer
-# from xblockutils.studio_editable import StudioEditableXBlockMixin, loader
-
+from xblockutils.resources import ResourceLoader
 
 class PrismXBlock(XBlock):
     """
     Provide syntax highlighting within a code editor 
     """
 
+    xblock_loader = ResourceLoader(__name__)
+
```

Comment:
I think it's better to use class constants so it's easy to be referenced in templates and other places.
```suggestion
        THEME_CHOICES = [
            {'display_name': 'light', 'value': 'light'},
            {'display_name': 'dark', 'value': 'light'}
        ]

        LANGUAGE_CHOICES = [
            {'display_name': 'python', 'value': 'python'},
            {'display_name': 'javascript', 'value': 'javascript'},
        ]
```


In the template it should be possible to use `{% for lang in self.LANGUAGE_CHOICES %}` once this is committed.

Reply:
brilliant!